packages/expect/src/types.ts

```diff
@@ -55,7 +55,8 @@ export type MatcherState = {
 export type AsymmetricMatcher = Record<string, any>;
 export type MatchersObject = {[id: string]: RawMatcherFn};
```

Comment:
Noticed while reviewing, are `expected` and `actual` reversed in the following definition?

```js
export type RawMatcherFn = (
  expected: any,
  actual: any,
  options?: any,
) => ExpectationResult
```

Reply:
nice catch - yes it is 🙂